apps/protocol-api/src/main.ts

```diff
@@ -338,4 +338,5 @@ app.get('/discord/oauth', async function (req, res) {
 });
 
 console.log('listening');
+console.log(process.env.NODE_ENV);
```

Comment:
did you add this for debugging purposes? just curious

Reply:
yep. was running aground on shield issues and wanted to make sure the proper env was being set